### ISSUE TITLE
Plexivity import IP addresses with ::ffff: prefix

### ISSUE THREAD

**Issue by fdbgbgsf** [Tautulli/Tautulli#730]
The plexivity database stores all IP addresses formatted as `<Player address="::ffff:192.168.178.12` and they are imported as such in PlexPy.

Any way to have that filtered out?

**Comment by JonnyWong16** [comment]
That's how Plex reports IPv4 mapped addresses.

**Comment by fdbgbgsf** [comment]
Sure, but that's not how PlexPy shows them, only the plays imported from Plexivity have this.

**Comment by JonnyWong16** [comment]
Oh, I did [remove](https://github.com/drzoidberg33/plexpy/blob/master/plexpy/pmsconnect.py#L1234) them for PlexPy. It's been a while since I did that so I didn't remember.

Change [this line](https://github.com/drzoidberg33/plexpy/blob/master/plexpy/plexivity_import.py#L111) to the following, then import your database again.

```
ip_address = helpers.get_xml_attr(d, 'address').split('::ffff:')[-1]
```

I'll fix it in the next update.

**Comment by fdbgbgsf** [comment]
Sweet. Thanks.

**Comment by JonnyWong16** [comment]
Fixed in v1.4.5.
